clang/test/CodeGenSYCL/add_ir_attributes_function.cpp

```diff
@@ -46,6 +46,16 @@ void
 free_func4() {
 }
 
+template <typename... Properties>
+#ifdef __SYCL_DEVICE_ONLY__
+[[__sycl_detail__::add_ir_attributes_function(
```

Comment:
Can you also please add a test with a mix of "" and names to ensure values for other names are still assigned correctly if "" is present?

Reply:
I have added such a test.